src/components/Sidebar/PriceBlock/PriceBlock.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from "react";
+import Slider from "react-slider";
+
+import SidebarSectionTitle from "../SidebarSectionTitle";
+
+import "./Price.scss";
+
+const MIN_VALUE = 1;
+const MAX_VALUE = 100;
+
+export const PriceBlock = () => {
+  const [values, setValues] = useState([MIN_VALUE, MAX_VALUE]);
+
+  return (
+    <div className="price-block">
+      <SidebarSectionTitle title="Price"/>
+      <Slider
+        className="price-block__slider"
+        thumbClassName="price-block__thumb"
+        trackClassName="price-block__track"
+        value={values}
+        onChange={setValues}
+        min={0}
+        max={1000}
+        pearling
+      />
+      <div className="price-block__inputs">
+        <label className="price-block__label">
+          Min
+          <input
+            type="number"
+            value={values[0]}
+            onChange={(e) =>
+              setValues([parseInt(e.target.value, 10), values[1]])
```

Comment:
Logic in JSX

Reply:
Removed onChange from inputs for now. It will be fully done in the logic branch.